plugins/DevicesDetection/Reports/GetType.php

```diff
@@ -27,7 +27,7 @@ protected function init()
 
     public function configureView(ViewDataTable $view)
     {
-        $view->config->show_search = false;
+        $view->config->show_search = true;
```

Comment:
Note: You could have also simply removed that line, as `true` is the default value. Only reports that don't want to use the search need to overwrite the value...